data/human/human missions.txt

```diff
@@ -3926,6 +3926,7 @@ mission "Mafia Extortion"
 
 mission "Street Swindle"
 	minor
+	invisible
```

Comment:
This mission never appears in your mission list, so I don't think making it `invisible` does anything.

Reply:
Yes, after you try two rounds. I looked at the file, and it says `accept` after each option on the second round. The alternative could be to make that `decline`.

Reply:
Good catch! We can't switch them to `decline` because that would negate the payment in the `on accept` section. Unfortunately, making it invisible isn't a good solution either - because the mission will just be active forever with no end. The resolution here is to use `action` in the conversation to apply the payment there and then make all outcomes `decline` (see more [here](https://github.com/endless-sky/endless-sky/wiki/WritingConversations)). Want to take a stab at it, @lumbar527 ? If not, I can probably make the change.

Reply:
I can handle that.

Reply:
I forgot, should I remove the `invisible` tag too, or is that fine there?

Reply:
Looking at the code to change it, `on accept` had a `fail` tag. I think the mission only showed up while you were landed (but it still should not have been visible).